src/Web/Program.cs

```diff
@@ -39,6 +39,17 @@
 builder.Services.Configure<ApiBehaviorOptions>(options =>
             options.SuppressModelStateInvalidFilter = true);
 
+builder.Services.AddCors(options =>
+{
+    options.AddDefaultPolicy(policy =>
+    {
+        policy.WithOrigins("http://localhost:44467")
```

Comment:
Should this be disabled in production mode? Sounds like we only care about this during development. Once we're in production mode we'll still be serving the Angular app from wwwroot. Couldn't allowing CORS from port 44467 in production lead to a security issue?

Reply:
I have removed this piece of code all together as it not needed anymore, we send the requestes from 44467 through `https://localhost:5309`